source/docs/3/solidwaste_mapping.rst

```diff
@@ -0,0 +1,252 @@
+
+   .. image:: /static/3/solidwaste_mapping/images/5.png
+      :align: center
+
+6. Once the filter is applied, only a subset of point will be visible on the map. Next we will add the ``waste_disposal_september2023.ods`` file. Click on the   :menuselection:`Open Data Source Manager` icon and switch to  :guilabel:`Vector` tab. Navigate the file by clicking on ... button given beside :guilabel:`File name`.  Click :guilabel:`Add`.
+
+   .. image:: /static/3/solidwaste_mapping/images/6.png
+      :align: center
+
+7. In the :guilabel:`Select Items to Add` dialog, select ``Disposal_per_Site_edited`` item and click :guilabel:`Add Layers``.
+
+   .. image:: /static/3/solidwaste_mapping/images/7.png
+      :align: center
+
+8. Open the attribute table of ``waste_disposal_september2023`` layer. This table has the name of the facility and total waste collected at the site for the month of September 2023.
+
+   .. image:: /static/3/solidwaste_mapping/images/8.png
+      :align: center
+
+9. Let’s join this table with the ``landfill_sites`` points layer. Go to :menuselection:`Processing --> Toolbox` from the menubar.
+
+   .. image:: /static/3/solidwaste_mapping/images/9.png
+      :align: center
+	  
+10. Search and locate the :guilabel:`Join attributes by Field Value` tool from the toolbox. Double-click to open it.
+
+   .. image:: /static/3/solidwaste_mapping/images/10.png
+      :align: center
+	  
+11. In the :guilabel:`Join Attributes by Field Value` dialog, select ``landfill_sites`` as the :guilabel:`Input layer` and ``NAME`` as the :guilabel:`Table field`. Select ``waste_disposal_september2023`` as the :guilabel:`Input layer 2` and ``Disposal Site`` as the :guilabel:`Table field 2`.
+Check the :guilabel:`Discard records which could not be joined` box. Save the :guilabel:`Joined layer` by clicking on ... button and select :guilabel:`Save to File`.
+
+   .. image:: /static/3/solidwaste_mapping/images/11.png
+      :align: center
+	  
+12. Name the output layer as ``waste_by_station_september_2023.shp`` and click :guilabel:`Run`.
+
+   .. image:: /static/3/solidwaste_mapping/images/12.png
+      :align: center
+	  
+13. Once the processing finishes, a new layer ``waste_by_station_september_2023`` will be added which will have the amount of waste in the ``Sept2023`` column.
+
+   .. image:: /static/3/solidwaste_mapping/images/13.png
+      :align: center
+	  
+14. Now let’s visualize this data. First select the ``Wards`` layer and click on :menuselection:`Open the Layer Styling panel` icon. 
+
+   .. image:: /static/3/solidwaste_mapping/images/14.png
+      :align: center
+
+15. Set the symbology of this layer to :guilabel:`Single Symbol` with a light :guilabel:`Fill color` and :guilabel:`Stroke color`.
+
+   .. image:: /static/3/solidwaste_mapping/images/15.png
+      :align: center
+
+16. Next select the ``waste_by_station_september_2023`` layer and select :guilabel:`Simple Marker` symbol. Click the drop-down for :guilabel:`Symbol layer type`.
```

Comment:
Screenshot shows 'wards' layer as active. Confusing for users. Make sure the active layer is the same as the one you are styling. Need to update all subsequent screenshots too.

Reply:
Done